lib/teamsnap/api.rb

```diff
@@ -40,7 +40,8 @@ def self.get_class(sym)
     def self.href(base_href, method, args = {})
       case method
       when :find, :delete
-        if [Fixnum, String].include?(args.class)
+        integer_class = 1.class
```

Comment:
should probably be a constant to reduce the call to `.class` each time an href is checked